combat-trainer.lic

```diff
@@ -4516,13 +4516,13 @@ class GameState
 
   # Determine the weapon skill to equip offhand to perform a doublestrike maneuver.
   def determine_doublestrike_skill
-    options = @aiming_trainables
+    options = @doublestrike_trainables
```

Comment:
Is it possible some players are relying on this behavior? I.e. they've configured/customized `aiming_trainables` for doublestrike purposes and haven't configured `doublestrike_trainables`?

Reply:
Good question, I've [asked](https://discord.com/channels/745675889622384681/745675890242879671/923284803736993803) in Discord for feedback on this change. Will follow up.

Reply:
On the positive side, any weapon listed in `doublestrike_trainables:` is allowable in both your main hand and offhand, even if you normally can't perform an _offhand_ attack with the weapons. Just tested on my Warrior Mage with a LB and LE. I could doublestrike with them but can't "attack left" with either.

The default config in base.yaml is:
```yaml
prioritize_maneuver_doublestrike: false

doublestrike_trainables:
  - Small Edged
  - Small Blunt
  - Large Edged
  - Large Blunt
 ```

If someone opts-in to doueblstrikes and only overrode `prioritize_maneuver_doublestrike: true`, then as long as their `weapon_training:` list includes 2 or more of those skills then they would still successfully doublestrike.